config/locales/en.yml

```diff
@@ -2490,6 +2490,7 @@ en:
   permission_save_queries: "Save views"
   permission_select_custom_fields: "Select custom fields"
   permission_select_project_modules: "Select project modules"
+  permission_share_work_packages: "Share work packages"
```

Comment:
Thank you, I missed this in the previous PR even though I ran into a bug with it before. It would be nice to have a spec file in core and each module that runs through all defined permissions and ensures that there is a translation present for them. I'll bring up a WorkPackage for it and assign it to myself

Reply:
Should be easy enough. We can iterate through all permissions with `OpenProject::AccessControl.permissions.map(&:name)` and then check that we have a translation for each one